heft-plugins/pre-compile-hardlink-or-copy-plugin/src/index.ts

```diff
@@ -45,6 +45,7 @@ export class PreCompileHardlinkOrCopyPlugin implements IHeftPlugin<IPreCompileHa
   }
 
   public readonly displayName: string = PLUGIN_NAME;
```

Comment:
```suggestion
```

Reply:
The RSC projects are still building using the old Heft which requires this property.